events/wc_primitive_events.txt

```diff
@@ -789,3 +793,166 @@ character_event = {
 		name = EVTOPTA_WCPRM_605
 	}
 }
+
+
+character_event = {
+    id = WCPRM.500
+    picture = GFX_evt_kobold_statute
+    desc = EVTDESC_WCPRM_500
+    border = GFX_event_normal_frame_religion
+	
+    is_triggered_only = yes
+	
+    immediate = {
+	    random_list = {
+		    20 = {
+			    character_event = {
+				    id = WCPRM.501 days = 7
+			    }
+		    }
+		    30 = {
+			    character_event = {
+			        id = WCPRM.502 days = 7
+			    }
+		    }
+			50 = {
+			    character_event = {
+				    id = WCPRM.503 days = 7
+				}
+			}
+	    }
+    }
+	
+	option = {
+	    name = EVTOPTA_WCPRM_500
+		piety = 1
+		wealth = -10
+	}
+}
+
+character_event = {
+    id = WCPRM.501
+	picture = GFX_evt_kobold_statute
+	desc = EVTDESC_WCPRM_501
+	border = GFX_event_normal_frame_religion
+	
+	
+	is_triggered_only = yes
+
+	immediate = {
+	    
+	}
+
+	option = {
+		name = EVTOPTA_WCPRM_501
+		piety = 50
+		
+		random_list = {
+		    10 = {
+				add_trait = ambitious
+				trigger = { NOT = { trait = ambitious } }
+			}
+			15 = {
+				add_trait = just
+				trigger = { NOT = { trait = just } }
+			}
+			10 = { 
+				add_trait = patient
+				trigger = { NOT = { trait = patient } }
+			}
+			15 = {
+				add_trait = diligent
+				trigger = { NOT = { trait = diligent } }
+			}
+			50 = { 
+			    prestige = 10
+			}
+		}
+	}
+}
+
+character_event = {
+    id = WCPRM.502
+	picture = GFX_evt_kobold_statute
+	desc = EVTDESC_WCPRM_502
+	border = GFX_event_normal_frame_religion
+	
+	
+	is_triggered_only = yes
+
+	immediate = {
+
+	}
+
+	option = {
+		name = ALAS
+		piety = -25
+		
+		random_list = {
+		    10 = {
+				add_trait = craven
+				trigger = { NOT = { trait = craven } }
+			}
+			15 = {
+				add_trait = envious
+				trigger = { NOT = { trait = envious } }
+			}
+			10 = { 
+				add_trait = wroth
+				trigger = { NOT = { trait = wroth } }
+			}
+			15 = {
+				add_trait = cruel
+				trigger = { 
+				    NOT = { 
+				        trait = cruel
+						trait = kind
+				}   }
+			}
```

Comment:
If you're kind, you can't become cruel? Is it intended? If it's, why you can't become craven if you're brave?

Reply:
Brave is opposite of craven, so it'll change. Kind is not opposite of cruel I think

Reply:
> Kind is not opposite of cruel I think
It's.